docs-new/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contribute to the Keptn documentation
+
+This document provides information about contributing to
+the [Keptn documentation](https://lifecycle.keptn.sh/docs/),
+which is part of the [Keptn](https://keptn.sh) website.
+
+The Keptn documentation is authored with
+[markdown](https://www.markdownguide.org/basic-syntax/)
+and rendered using
+[MkDocs](https://www.mkdocs.org/).
+
+We welcome and encourage contributions of all levels.
+You can make modifications using the GitHub editor;
+this works well for small modifications but,
+if you are making significant changes,
+you may find it better to fork and clone the repository
+and make changes using the text editor or IDE of your choice.
+You can also run the website locally
+to check the rendered documentation
+and then push your changes to the repository as a pull request.
+
+If you need help getting started,
+feel free to ask for help on the `#help-contributing` or `#keptn-docs` channels on the [Keptn Slack](https://keptn.sh/community/#slack).
```

Comment:
Let's replace this with the CNCF #keptn channel since the keptn slack will be deleted soon.
```suggestion
feel free to ask for help on the `#keptn` channel on the [CNCF Slack](https://cloud-native.slack.com).
```